lib/import-export-visitor.js

```diff
@@ -276,11 +277,15 @@ module.exports = class ImportExportVisitor extends Visitor {
     if (this.code) {
       const oldLines = this.code.slice(oldStart, oldEnd).split("\n");
       const newLines = newCode.split("\n");
+      const line = oldLines[0];
+      const endCode = line.charCodeAt(line.length - 1);
+      const lineEnd = endCode === carrageReturnCode ? "\r\n" : "\n";
```

Comment:
The goal of this code is basically to blank out the lines where the `import` or `export` declarations originally appeared, so really what we should be doing is just removing everything from those lines except the line ending. If we did that right, we could just preserve the original line endings, instead of using `oldLines[0]` to detect whether all the lines should end with `\r\n` or `\n`.